app/models/listing.rb

```diff
@@ -52,7 +53,7 @@ def name
   end
 
   def name_and_match_history
-    "(#{type}-#{all_tags_to_s.upcase})</strong> #{person.match_history}. (#{person.name})"
+    "(#{type}-#{all_tags_to_s.upcase}#{" ***INEXHAUSTIBLE*** " if inexhaustible?}) #{person.match_history}. (#{person.name}#{" --- $" + person.monthly_donation_amount_max.to_s + "/mo left" if person.monthly_donation_amount_max&.to_f > 0.0})"
```

Comment:
Lot going on on this line :)

Reply:
lol. yup. it's pretty painful. the page this method is called from will be replaced once we add an official "match" page (a version of the Contributions page), so we can get rid of this monstrosity then.